spanner/hibernate/pom.xml

```diff
@@ -6,13 +6,13 @@
   <modelVersion>4.0.0</modelVersion>
 
   <properties>
-    <maven.compiler.target>1.8</maven.compiler.target>
```

Comment:
can you please add some context in the PR description around why we're upgrading this to Java 11, and why we're removing the spanner dependency? Thanks!

Reply:
Hibernate 6 requires Java 11, see [here](https://docs.jboss.org/hibernate/orm/current/userguide/html_single/Hibernate_User_Guide.html#_system_requirements).

Also, spanner dialect has been moved inside hibernate package, i.e., `org.hibernate.dialect.SpannerDialect`.